src/app/clients/ActionBar.tsx

```diff
@@ -0,0 +1,188 @@
+"use client";
+
+import supabase, { Schema } from "@/supabase";
+import React, { useState } from "react";
+import ActionBarModal from "@/app/clients/ActionBarModal";
+import styled from "styled-components";
+import Button from "@mui/material/Button/Button";
+import Menu from "@mui/material/Menu/Menu";
+import MenuList from "@mui/material/MenuList/MenuList";
+import MenuItem from "@mui/material/MenuItem/MenuItem";
+import { Dayjs } from "dayjs";
+
+interface Props {
+    selected: number[];
+    data: Schema["parcels"][];
+}
+
+const statuses = [
+    "Request Denied",
+    // probably shouldn't be a status (?)
+    "No Status",
```

Comment:
Idk if this would make more sense as `"Reset Status"` if that is what is meant to be? If `No Status` is what it's meant to be that's fine though

Reply:
"No Status" was copied from their current implementation, the comment was an expression of my own confusion